static/coffee/controllers.coffee

```diff
@@ -573,12 +588,12 @@ controllers.controller('CaseController', ['$scope', '$window', '$timeout', 'Case
   #----------------------------------------------------------------------------
 
   $scope.onWatch = () ->
-    UtilsService.confirmModal("Receive notifications for activity in this case?").then(() ->
+    UtilsService.confirmModal("Receive alerts for activity in this case?").then(() ->
```

Comment:
Woops, this was an accidental replace from a search-and-replace, sorry! Will fix.

> Also can we replace these calls with ModalService.confirm(..) calls so we're certain that new service can replicate the existing functionality, and we don't have two confirm modal methods.

Can replace these calls here, but I'm a bit worried about replacing all confirm modals in this PR, get the feeling it will make this PR's scope grow quite a bit.